base/templates/app.html

```diff
@@ -46,6 +46,9 @@
         {% fiduswriter_config_js %}
         <!-- Main page script -->
         <script type="text/javascript" src="{% static "js/app.mjs" %}"></script>
+
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/cropperjs/1.4.3/cropper.min.css" />
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/cropperjs/1.4.3/cropper.min.js"></script>
```

Comment:
We need to replace these with links to our local copy of the CSS file. And we shouldn't import cropper.min.js at all given that we import it directly in the JavaScript file.

Reply:
To get the local version of the CSS file, you will probably need to copy it out of the node_modules folder in a script inside the package.json file. Look at how it's done for fontawesome in base/package.json. I believe if you create a script called `copy_croppedjs_css` and one called `post_install` with value 'npm run copy_croppedjs_css' then it will merge the `post_install` scripts correctly together.